project/plugins.sbt

```diff
@@ -1,10 +1,12 @@
-resolvers += MavenRepository("HMRC-open-artefacts-maven2", "https://open.artefacts.tax.service.gov.uk/maven2")
-resolvers += Resolver.url("HMRC-open-artefacts-ivy", url("https://open.artefacts.tax.service.gov.uk/ivy2"))(Resolver.ivyStylePatterns)
+resolvers += "HMRC-open-artefacts-maven2" at "https://open.artefacts.tax.service.gov.uk/maven2"
+resolvers += Resolver.url("HMRC-open-artefacts-ivy", url("https://open.artefacts.tax.service.gov.uk/ivy2"))(
+  Resolver.ivyStylePatterns
+)
 
-addSbtPlugin("uk.gov.hmrc"       % "sbt-auto-build"         % "3.14.0")
-addSbtPlugin("uk.gov.hmrc"       % "sbt-distributables"     % "2.2.0")
-addSbtPlugin("com.typesafe.play" % "sbt-plugin"             % "2.8.20")
+addSbtPlugin("uk.gov.hmrc"       % "sbt-auto-build"         % "3.20.0")
+addSbtPlugin("uk.gov.hmrc"       % "sbt-distributables"     % "2.5.0")
+addSbtPlugin("org.playframework" % "sbt-plugin"             % "3.0.2")
 addSbtPlugin("com.eed3si9n"      % "sbt-assembly"           % "0.15.0")
```

Comment:
This version is very old, Can you try if the latest 2.2.0 version works fine. Or any newer versions

Reply:
updated